### DISCUSSION TITLE
custom class for MultiSelect component

### DISCUSSION THREAD

**Discussion by ArgoZhang** [dotnetcore/BootstrapBlazor#3548] [Q&A] [answered]
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/22001478/cd2bf2fd-07c9-4e93-a790-1f106511ab8e)

@anna777x 

hi, could you take a look at this?

the `CustomClass` parameter used for `dropdown-toggle` HTML element. it's designed by `bootstrap`.

so what are your clear needs? why add a class for the root element?

**Comment by anna777x** [comment]
yes, you're absolutely right. I thought that custom class should apply to parent element, but I didn't notice that it applies it to next element.

**Reply by anna777x** [reply]
I've closed that PR. so I'm coming back to opening/closing modals)

**Reply by ArgoZhang** [reply, marked as answer]
@anna777x  if you wanna modify the root element style. you can use CSS variable

```
.multi-select {
    --bb-multi-select-min-height: 35px;
    --bb-multi-select-max-height: 65px;
    --bb-multi-select-button-bg-color: rgba(var(--bs-body-color-rgb), .15);
    --bb-multi-select-button-hover-bg-color: rgba(var(--bs-body-color-rgb), .3);
    --bb-multi-select-item-margin-x: 3px;
    --bb-multi-select-item-margin-y: 3px;
    --bb-multi-select-item-padding: 2px 6px;
    --bb-multi-select-item-max-width: 130px;
}
```

you can find it on the following link 

https://github.com/dotnetcore/BootstrapBlazor/blob/main/src/BootstrapBlazor/Components/Select/MultiSelect.razor.scss

the variable value will be transformed to `https://github.com/dotnetcore/BootstrapBlazor/blob/main/src/BootstrapBlazor/wwwroot/scss/theme/bootstrap.blazor.scss`

for theme
